### DISCUSSION TITLE
I found a solution!!!1!!1

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#374] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Step 1: find a job or something!
Step 2: work and save money!
Step 3: buy a laptop!!!1!
done!!! no moar blocks!

**Comment by Aka-but-nice** [comment]
LES GO IT WORKED THANKS SO MUCH

**Reply by Aka-but-nice** [reply]
YOUR SO HELPFUL, look at me tryna make ingot for 106 and higher, like this is truely ingot for 106 and higher I can not give more credit

**Comment by rwg100** [comment]
This is one of the answers ever made.
It includes steps, punctuation, and not to mention, words!
This truly is one of the answers ever created.

**Reply by Aka-but-nice** [reply]
Give this awesome man the noble peace prize
